agent/lib/src/health.dart

```diff
@@ -66,6 +66,7 @@ Future<AgentHealth> performHealthChecks(Agent agent) async {
             } else {
               results['able-to-build-and-sign'] = HealthCheckResult.failure(
                   'Failed to build and sign iOS app.');
+              await getFlutterAt('master');
```

Comment:
Sometimes we need to run agent against a specific commit, I'm afraid this line would defect that usage.

Reply:
This should not affect when running the test using run and using the revision parameter. Also this will run only if the health check fails.

The only way this could affect is if we want to run a test with whatever is already available on $HOME/.cocoon.

Reply:
LGTM for this change. I do feel the health check is pretty heavy though.

Reply:
Let's evaluate if it is still needed and if not we can remove it.